electron-builder.json

```diff
@@ -109,9 +109,6 @@
       "NSCameraUsageDescription": "Camera access may be used by Mattermost plugins, such as Jitsi video conferencing.",
       "NSFocusStatusUsageDescription": "Focus status is used by Mattermost to determine whether to send notifications or not.",
       "LSFileQuarantineEnabled": true
-    },
-    "notarize": {
```

Comment:
Just curiosity, what did this used to do?

Reply:
Just specified the team id as part of the process. It's not a secret so we just put it there instead of in an env variable.